syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -984,6 +984,15 @@ def set_garbage_collect_data(self, value):
         for _, share in shares.items():
             share.garbage_collect_data = value
 
+    def get_garbage_collect_data(self):
```

Comment:
This seems like a good addition, but doesn't look like it actually gets called anywhere yet. Did I miss it?

Reply:
It was used in how previously garbage_collect_data was handled.